feature-group-definitions/temporal.yml

```diff
@@ -1,3 +1,4 @@
 name: Temporal
+description: The Temporal API provides a way to work with dates and times in JavaScript.
```

Comment:
I like the comparison with `Date`, since someone might be confused about why there are two ways to deal with Dates. However, I think I prefer the `The X thing does Y` than `X is a thing for Y`. It feels more consistent with the other descriptions we have.

```suggestion
description: The `Temporal` API allows you to work with dates, times, time zones, and durations. It is more powerful than the `Date` API.
```

Reply:
That's a better description all around, let's go with that!